### PR TITLE
Draft release notes

### LINKED ISSUE #2095: QA Day: Release notes

### PR DESCRIPTION
**Pull request by RudolfWeeber**
Fixes #2095

### CODE REVIEW

File: NEWS [hunk 1]
```diff
@@ -2,42 +2,53 @@
...
+TCL-based versions.
+
+Transitioning from TCL-based versions
+-------------------------------------
+
+The following can serve as a starting point for the new interface: 
+* The "Introduction" chapter in the user's guide
+* The tutorials in the /doc/tutorials directory of the distribution.
+
+Please note that, starting from ESPResSo 4.0, enabling additoinal features in myconfig.hpp no longer changes the behavior of a simulation scripts. Features such as particle rotation and virtual sites have to be activated explicitly in the simulation scripts.
```
Comment (KaiSzuttor) [inline NEWS]: typo

File: NEWS [hunk 2]
```diff
@@ -2,42 +2,53 @@
...
+
+New features:
+-------------
+* Reaction ensemble method for the modelling of chemical reactions
+* Polarisable particles via the Drude-oscillator scheme
+* Steepest-descent energy minimization scheme 
+* Methods for active particles (swimmer_reaction, dipolar swimmers)
+* GPU-accererated P3M method for electrostatic interactions
+* GPU-accelerated direct summation of dipolar interactoins under open boundary conditions
```
Comment (KaiSzuttor) [inline NEWS]: typo

File: NEWS [hunk 3]
```diff
@@ -2,42 +2,53 @@
...
-* bit_random tcl command removed.
 
-Changes visible for developers
-------------------------------
-* CMake build system
+* When the ROTATION feature is compiled in, the rotation of a particle around each of its Cartesian axes in the body-fixed frame can be enabled
+* The rotational degrees of freedom of virtual sites is now integrated. Virtual sites no longer use the quaternion attributes to store their relative orientation
+* Alternatively, the orientation of the virtual site can be specified relative to that of the particle it is derived from.
+* The random number generator was changed to Mersenne Twister as implemented by C++11.
+* Espresso now depends on the Boost library including Boost-Mpi and a C++11 capable compiler
```
Comment (KaiSzuttor) [inline NEWS:53]: didn't we want to switch to c++14?

**Review by KaiSzuttor — CHANGES_REQUESTED** [review]

**Comment by fweik** [comment]
Yes, but not for the release.

File: NEWS [hunk 4]
```diff
@@ -2,42 +2,73 @@
...
+* The build system is now based on CMake.
+
+Removed featues
+---------------
+The following features will likely not be part of the ESPResSo 4.0 release, but support is expected to be re-added in a subsequent release.
+* Generalized hybrid Monte Carlo thermostat
+* Virtual sites located at the center of mass of a group of particles
+* Metadynamics, umbrella sampling and parallel tempering
+* Non-equilibrium molecular dynamics (shear boundary conditions implemented by moving slabs of particles at the boundaries)
+* The memd/meggs electrostatic solver
```
Comment (KaiSzuttor) [inline NEWS]: typo maggs

File: NEWS [hunk 5]
```diff
@@ -2,42 +2,73 @@
...
+* The com_force feature
+* The Chan Shen lattice Boltzmann extension
```
Comment (KaiSzuttor) [inline NEWS]: this is still in for GPU LB

File: NEWS [hunk 6]
```diff
@@ -2,42 +2,73 @@
...
+
```
Comment (KaiSzuttor) [inline NEWS]: what about lees edwards?

File: NEWS [hunk 7]
```diff
@@ -2,42 +2,73 @@
...
+The following functionality is removed permanently:
+* Blockfile support, which is replaced by support for the h5md format as well as checkpointing based on the pickle Python module.
+* Some of the convenience functions for particle creation such as salt and counterions, as these can be replaced by a very few lines of Python in the simulation script.
+* The lj_angle and angledist (not-qutie)-pair potentials
```
Comment (KaiSzuttor) [inline NEWS]: typo?

File: NEWS [hunk 8]
```diff
@@ -2,42 +2,73 @@
...
+* The following analysis methods:
+  diffusion_profile, cwvac, p_isnt, ..._mol, cel_gpb, dipmom_normal, MSd, angularmomentum, cluster_size_dist, mol, lipid_orient_order, get_lipid_oriends, current, wall_stuff,
```
Comment (KaiSzuttor) [inline NEWS]: typo: p_inst, MSD, get_lipid_orients

**Review by KaiSzuttor — DISMISSED** [review]

**Comment by RudolfWeeber** [comment]
LE was never part of an official release, afaik. Mentioned it anyway, now.
Gpu Shan Chen has no (finished) Python interface. We only test that it still builds.

File: NEWS [hunk 9]
```diff
@@ -2,42 +2,74 @@
...
+
+The following can serve as a starting point for the new interface: 
+* The "Introduction" chapter in the user's guide
+* The tutorials in the /doc/tutorials directory of the distribution.
+
+Please note that, starting from ESPResSo 4.0, enabling additional features in myconfig.hpp no longer changes the behavior of a simulation scripts. Features such as particle rotation and virtual sites have to be activated explicitly in the simulation scripts.
+
+New features:
+-------------
+* Reaction ensemble method for the modelling of chemical reactions
```
Comment (jonaslandsgesell) [inline NEWS]: The constant pH method is also implemented and differs in the acceptance probability. I think it is worth to note both methods here.

**Review by jonaslandsgesell — CHANGES_REQUESTED** [review]

**Comment by RudolfWeeber** [comment]
IBM and Object in Fluid

**Comment by RudolfWeeber** [comment]
@fweik, from my side, everything is in. Merge, if you are happy with it.

**Comment by fweik** [comment]
LGTM except for the two typos that kai pointed out.

File: NEWS [hunk 10]
```diff
@@ -2,42 +2,76 @@
...
-------------------------------
-* CMake build system
+* When the ROTATION feature is compiled in, the rotation of a particle around each of its Cartesian axes in the body-fixed frame can be enabled
+* The rotational degrees of freedom of virtual sites is now integrated. Virtual sites no longer use the quaternion attributes to store their relative orientation
+* Alternatively, the orientation of the virtual site can be specified relative to that of the particle it is derived from.
+* The random number generator was changed to Mersenne Twister as implemented by C++11.
+* Espresso now depends on the Boost library including Boost-Mpi and a C++11 capable compiler
+* The build system is now based on CMake.
+
+Removed featues
```
Comment (KonradBreitsprecher) [inline NEWS]: Typo featues

File: NEWS [hunk 11]
```diff
@@ -2,42 +2,77 @@
...
+Removed features
+----------------
+The following features are not part of the ESPResSo 4.0 release, but support is expected to be re-added in a subsequent release.
+* Generalized hybrid Monte Carlo thermostat
+* Virtual sites located at the center of mass of a group of particles
+* Metadynamics, umbrella sampling and parallel tempering
+* Non-equilibrium molecular dynamics (shear boundary conditions implemented by moving slabs of particles at the boundaries)
+* The memd/maggs electrostatic solver
+* The com_force feature
+* The Chan Shen lattice Boltzmann extension
```
Comment (hmenke) [inline NEWS]: ~~Chan Shen~~ Shan Chen

File: NEWS [hunk 12]
```diff
@@ -2,42 +2,105 @@
...
+* Changing interaction parameters does not automatically update the value of 
+  the force property stored on the particles. Forces are re-calculated in
+  the next integration step. (#1973)
+* The domain decomposition cell system does not always choose the node grid
+  yielding the best performance in parallel simulations (#1970)
+* The dipolar P3M method can only be tuned once in a simulation script,
+  unless method parameters are explicitly reset to 0 before the second tuning 
+  (#1937)
+* The chain-analysis routines (such as rdfchains) fail the test case in 
+  about 1% of the time and leak memory. (#2187, 2186)
```
Comment (fweik) [inline NEWS]: The leak has been fixed. The failures still exist.

**Review by fweik — APPROVED** [review]

**Pull request merged**